client/src/pages/OnboardingPage/PersonalSettingsForm/PersonalSettingsForm.js

```diff
@@ -16,16 +16,59 @@ class PersonalSettingsForm extends Component {
         dream_destination: "",
         first_name: this.props.setup.first_name,
         last_name: this.props.setup.last_name,
-        city_of_residence: this.props.filter.city
-      }
+        city_of_residence: this.props.filter.city,
+      },
     };
   }
+  getSignedRequest = (file) => {
```

Comment:
This method is something that would go into the service for users (can chat about that later)

Reply:
Will pause on that until that folder gets create by Nate